puppet/modules/consul/.fixtures.yaml

```diff
@@ -0,0 +1,7 @@
+fixtures:
```

Comment:
The whole file should be called .fixtures.yml and you should have the exact same file in `.fixtures.yml.local`